hiredis.h

```diff
@@ -46,8 +46,8 @@ typedef long long ssize_t;
 #include "alloc.h" /* for allocation wrappers */
 
 #define HIREDIS_MAJOR 1
-#define HIREDIS_MINOR 1
-#define HIREDIS_PATCH 1
+#define HIREDIS_MINOR 2
+#define HIREDIS_PATCH 0
 #define HIREDIS_SONAME 1.1.1-dev
```

Comment:
We want to maintain the same `SONAME` as exists in v1.1.0

```suggestion
#define HIREDIS_SONAME 1.1.0
```

Reply:
I realized this may not be perfectly clear.  We just need to make sure the `SONAME` is unchanged from `1.1.0` in the actual tag commit.